spec.md

```diff
@@ -752,7 +757,12 @@ A Controller Plugin MUST implement this RPC call if it has `CREATE_DELETE_VOLUME
 This RPC will be called by the CO to provision a new volume on behalf of a user (to be consumed as either a block device or a mounted filesystem).
 
 This operation MUST be idempotent.
-If a volume corresponding to the specified volume `name` already exists, is accessible from `accessibility_requirements`, and is compatible with the specified `capacity_range`, `volume_capabilities` and `parameters` in the `CreateVolumeRequest`, the Plugin MUST reply `0 OK` with the corresponding `CreateVolumeResponse`.
+If a volume corresponding to the specified volume `name` already exists, is accessible from `accessibility_requirements`, and is compatible with the specified `capacity_range`, `volume_capabilities`, `parameters` and `mutable_parameters` in the `CreateVolumeRequest`, the Plugin MUST reply `0 OK` with the corresponding `CreateVolumeResponse`.
+
+The `parameters` field SHALL contain opaque volume attributes to be specified at creation time. 
+This field SHOULD be used to specify features of the volume that SHOULD NOT change. 
```

Comment:
This contradicts earlier guidance so we need to be careful here. SHOULD is gentler than MUST but possibly not gentle enough because many SPs are likely to continue allowing options which can change in this field forever as a backwards compatibility measure. Possibly just remove this line,

Reply:
Okay I changed it to:

```
This field MUST be used to specify features of the volume that MUST NOT change. 
```